celery/backends/asynchronous.py

```diff
@@ -107,6 +102,11 @@ def spawn(self):
         from eventlet import spawn
         return spawn
 
+    def wait_for(self, p, wait, timeout=None):
```

Comment:
The `wait` argument is only used for the non-greenlet (non-gevent, non-eventlet) flow.  Sort of confusing to have it around here.